app/pages/reservationSearch/day-reservations/reservation-row/ReservationRow.spec.js

```diff
@@ -48,4 +49,22 @@ describe('pages/reservationSearch/reservation-row/ReservationRow', () => {
     const eventSubject = getWrapper().find('.event-subject');
     expect(eventSubject.text()).to.equal(defaults.eventSubject);
   });
+
+  it('renders a number of participants icon', () => {
```

Comment:
- [ ] There was already a test "renders number of participants" (which only tests the number, not icon). You could remove that other one now.